### PR TITLE
@eessex => [EOY] QA, Responsive Image Sets, Super Article Nav Updates

### PR DESCRIPTION
**Pull request by kanaabe**
- Fixes Hamburger nav not expanding on mobile
- Hides Edit button on mobile
- Responsive Image Set
- Dynamic Carousel (section 4)
- Super article nav updates
- Placeholder image for bg video

### CODE REVIEW

File: apps/editorial_features/components/eoy/stylesheets/body.styl [hunk 1]
```diff
@@ -41,6 +41,8 @@ img
   &__content
     max-width 1750px
     margin 0 auto
+    p a
+      fine-faux-underline()
```
Comment (eessex) [inline apps/editorial_features/components/eoy/stylesheets/body.styl]: yay!

File: apps/editorial_features/components/eoy/templates/index.jade [hunk 2]
```diff
@@ -9,19 +9,20 @@ block head
 
 block body
   - var superArticle = article
-  include ../../../../../components/article/templates/super_article_sticky_header.jade
+  .eoy-feature__sa-header
+    include ../../../../../components/article/templates/super_article_sticky_header.jade
 
   .eoy-feature
-    video(class='video' autoplay=true, controls=false, loop=true, muted=true, playsinline=true).eoy-feature__background.active
+    video(class='video' autoplay=true, controls=false, loop=true, muted=true, playsinline=true poster=curation.get('placeholder')).eoy-feature__background.active
```
Comment (eessex) [inline apps/editorial_features/components/eoy/templates/index.jade]: is this a still frame?

Comment (kanaabe) [inline apps/editorial_features/components/eoy/templates/index.jade]: yep it's the fallback for people without html5 video on their browser.

**Comment by eessex** [comment]
Looking great, can't wait to see the sub articles in their final form!

File: apps/editorial_features/components/eoy/templates/index.jade (hunk 2 above)
Comment (kanaabe) [inline apps/editorial_features/components/eoy/templates/index.jade]: Hmm yes good idea, but we'll want to do different images for those probably. I'll make a note in the issue tracker and mark it as "nice to have". thank you!

File: apps/editorial_features/components/eoy/client.coffee [hunk 3]
```diff
@@ -41,7 +41,6 @@ module.exports.EoyView = class EoyView extends Backbone.View
         watchScrolling()
     $(window).resize =>
       @videoControls()
-      @setupCarousel()
```
Comment (kanaabe) [inline apps/editorial_features/components/eoy/client.coffee]: I removed this since we only need to initialize the carousel once!

Comment (eessex) [inline apps/editorial_features/components/eoy/client.coffee]: I was noticing that the carousel height is set on initialize at 600 and stays there on resize -- were you able to get this responsive?

Comment (kanaabe) [inline apps/editorial_features/components/eoy/client.coffee]: yep!

**Pull request merged**